packages/edit-site/src/components/sidebar/global-styles-v2/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { __experimentalNavigation as Navigation } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { GLOBAL_STYLES_VIEWS, VIEW_ROOT } from './navigation/constants';
+import GlobalStylesNavigation from './navigation';
+import GlobalStylesCurrentView from './navigation/view';
+
+export default function GlobalStylesV2() {
+	return (
+		<GlobalStylesNavigation>
+			<GlobalStylesView />
+		</GlobalStylesNavigation>
+	);
+}
+
+function GlobalStylesView() {
+	return (
+		<Navigation data={ GLOBAL_STYLES_VIEWS } initial={ VIEW_ROOT }>
```

Comment:
It seems like we're "defaulting" to `VIEW_ROOT` in 2 places (here and when setting the initial value of `currentView` in `packages/edit-site/src/components/sidebar/global-styles-v2/navigation/index.js`)

Reply:
Also, I'm not very familiar with this component, but after a quick look I can't see the `data` and `initial` props being defined in `packages/components/src/navigation/index.js`

Reply:
> It seems like we're "defaulting" to `VIEW_ROOT` as the initial view in 2 places (here and when setting the initial value of `currentView` in `packages/edit-site/src/components/sidebar/global-styles-v2/navigation/index.js`), should we aim at having it defined only in one place as the initial/starting view?

I like that idea 👍 

> Also, I'm not very familiar with this component, but after a quick look I can't see the `data` and `initial` props being defined in `packages/components/src/navigation/index.js`

I think these are remnants of a previous approach that I didn't clean up. Can be removed.